proto/backup.proto

```diff
@@ -43,6 +45,9 @@ message BackupMeta {
 
     // In incremental backup, DDLs which are completed in (lastBackupTS, backupTS] will be stored here.
     bytes ddls = 10;
+
+    // Encryption method and key.
+    encryptionpb.EncryptionConfig encryption = 11;
```

Comment:
what is the purpose of this config here?

if the backupmeta is in plaintext the key will be exposed. if the backupmeta is encrypted this field is redundant.

Reply:
backupmeta is encrypted using aes256-gcm (unconfigurable). data is encrypted using the config provided by user and stored here. I'm still debating with myself whether the two encryption should have different key - as they may be of different key length, and truncating a key doesn't sound right.

I figure I may need some document, as the implementation I got is not super straightforward. I'll send a RFC soon.

Reply:
Given that we're going to use HKDF, please change this to 

```suggestion
     encryptionpb.EncryptionMethod encryption_method = 11; 
```